cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -214,8 +216,8 @@ log INFO "== Kubernetes addon manager started at $(date -Is) with ADDON_CHECK_IN
 token_found=""
 while [ -z "${token_found}" ]; do
   sleep .5
-  token_found=$(${KUBECTL} ${KUBECTL_OPTS} get --namespace="${SYSTEM_NAMESPACE}" serviceaccount default -o go-template="{{with index .secrets 0}}{{.name}}{{end}}")
-  if [[ $? -ne 0 ]]; then
+  token_found=$(${KUBECTL} "${KUBECTL_OPTS}" get --namespace="${SYSTEM_NAMESPACE}" serviceaccount default -o go-template="{{with index .secrets 0}}{{.name}}{{end}}")
+  if [[ $? ]]; then
```

Comment:
Can these two lines be combined with an `if !`?

Reply:
Thanks for your feedback @fejta .
But I think that's OK now.
Because the `token_found` is not equal to `$?` in this scenario.